apps/console/src/features/validation/api/validation-config.ts

```diff
@@ -205,3 +218,60 @@ const prepareValidationConfigData = (values: ValidationFormInterface): Validatio
         rules: rules
     };
 };
+
+const prepareUsernameValidationConfigData = (values: ValidationFormInterface): ValidationDataInterface => {
+ 
+    if(values.field=="password"){
+        return;
+
+    }
+
+    const rules: ValidationConfInterface[] = [];
+
+    if (values.enableValidator=="false"){
+        rules.push(
+            {
+                properties: [
+                    {
+                        key: "enable.validator",
+                        value: "true"
+                    }
+                ],
+                validator: "EmailFormatValidator"
+            }
+        );
+        
+    } else {
+        rules.push(
+            {
+                properties: [
+                    {
+                        key: "enable.validator",
+                        value: "true"
+                    }
+                ],
+                validator: "AlphanumericValidator"
+            }
+        );
+        rules.push(
+            {
+                properties: [
+                    {
+                        key: "min.length",
+                        value: values.minLength ? values.minLength : "3"
+                    },
+                    {
+                        key: "max.length",
+                        value: values.maxLength ? values.maxLength : "50"
+                    }
+                ],
+                validator: "LengthValidator"
+            }
+        );
```

Comment:
We can use a single `push` to append both rules to the array, can't we?

eg: 

```
const arr: Array = []
arr.push(obj1, obj2,obj3)
console.log(arr)
> [obj1,obj2,obj3]
```

Reply:
Fixed by c2507cb1bbad939ec519a8012f3bc2a1abf8cc73.